Course_Part_09/src/Assignment41.java

```diff
@@ -0,0 +1,38 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class Assignment41 {
+    public static void main(String[] args) {
+
+        String fileName = "dummy2.txt";
+        // Classic BufferedReader and FileReader using try-with-resources construct
+        try (FileReader reader = new FileReader("Course_Part_09/src/" + fileName);
+             BufferedReader bufferedReader = new BufferedReader(reader)) {
+
+            int wordCount = 0;
+            int upperCount = 0;
+            // read line by line
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] words = line.split(" ");
+                for (String word : words) {
+                    wordCount++;
```

Comment:
Goede uitwerking 👍 

Wat ook kan is om het tellen van de woorden buiten de for loop te houden door `words.length` te gebruiken. `words.length` bevat immers per regel het aantal woorden. dan krijg je tussen regel 18 en 19 de volgende code:

```
wordCount += words.length;
```